backend/src/main/java/com/festago/auth/application/AdminAuthService.java

```diff
@@ -0,0 +1,81 @@
+package com.festago.auth.application;
+
+import com.festago.auth.domain.Admin;
+import com.festago.auth.domain.AdminRepository;
+import com.festago.auth.domain.AuthPayload;
+import com.festago.auth.domain.AuthProvider;
+import com.festago.auth.domain.Role;
+import com.festago.auth.domain.SocialType;
+import com.festago.auth.dto.AdminLoginRequest;
+import com.festago.auth.dto.AdminSignupRequest;
+import com.festago.auth.dto.AdminSignupResponse;
+import com.festago.domain.Member;
+import com.festago.exception.BadRequestException;
+import com.festago.exception.ErrorCode;
+import com.festago.exception.UnauthorizedException;
+import java.util.Objects;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class AdminAuthService {
+
+    private static final String ADMIN = "admin";
+
+    private final AuthProvider authProvider;
+    private final AdminRepository adminRepository;
+
+    public AdminAuthService(AuthProvider authProvider, AdminRepository adminRepository) {
+        this.authProvider = authProvider;
+        this.adminRepository = adminRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public String login(AdminLoginRequest request) {
+        Admin admin = findAdmin(request);
+        validatePassword(admin.getPassword(), request.password());
+        AuthPayload authPayload = getAuthPayload(admin);
+        return authProvider.provide(authPayload);
+    }
+
+    private Admin findAdmin(AdminLoginRequest request) {
+        return adminRepository.findByUsername(request.username())
+            .orElseThrow(() -> new UnauthorizedException(ErrorCode.INVALID_PASSWORD));
+    }
+
+    private void validatePassword(String password, String comparePassword) {
+        if (!Objects.equals(password, comparePassword)) {
+            throw new UnauthorizedException(ErrorCode.INVALID_PASSWORD);
+        }
+    }
+
+    private AuthPayload getAuthPayload(Admin admin) {
+        Long memberId = admin.getMemberId();
+        return new AuthPayload(memberId, Role.ADMIN);
```

Comment:
마찬가지로 Member에 Role 컬럼이 생긴다면 다음과 같이 될 것 같네요.
```suggestion
        Role role = admin.getRole(); // admin.getMember().getRole(); or account.getMember().getRole();
        return new AuthPayload(memberId, role);
```